app/Repositories/IndependentActivity/IndependentActivityRepository.php

```diff
@@ -966,7 +967,7 @@ public function copyToPlaylist($independentActivity, $playlist, $token)
             'order' => $this->getOrder($playlist->id) + 1,
             'h5p_content_id' => $newH5pContent, // set if new h5pContent created
             'thumb_url' => $new_thumb_url,
-            'shared' => 0,
+            'shared' => Project::where('id', $playlist->project_id)->value('shared'),
```

Comment:
The "shared" for the project can be get using following code

$playlist->project->shared